### ISSUE TITLE
Is the system completely free

### ISSUE THREAD

**Issue by blueprintparadise** [exadel-inc/CompreFace#600]
Is the system based on a freemium model or is it free indefinitely from start to finish?

**Comment by pospielov** [comment]
The current version (0.6) is distributed under Apache 2.0 license, so it's totally free with no limitations for any functionality. You can use it in commercial projects as well.
At this point, our plan is to use a typical open-source monetization strategy, which is a free product with paid support and services. The offered services include integration, customization, new feature development, training, etc.
So, there is no paid version of CompreFace for now and there are no plans for it. But I can't guarantee that there won't be such plans in the future.

**Comment by bobmoff** [comment]
does this also include the pretrained models used ?

**Comment by pospielov** [comment]
We didn't create models for facial recognition. So we do not control their license. 
Still, all libraries we use were taken from open sources. 
E.g. FaceNet library is distributed by MIT license, with no information about the license of the model, so we expect their model is also distributed by this license. 
There potentially could be a problem with InsightFace models. When we downloaded them, they were distributed with an MIT license. When you install CompreFace, you use the model we downloaded by MIT license. However, now InsightFace says they do not allow to use of models for commercial purposes. I'm not a lawyer, so I can't say if the license could be changed for all copies previously downloaded.
